control.proto

```diff
@@ -244,5 +244,5 @@ service ElasticAgentControl {
   rpc DiagnosticAgent(DiagnosticAgentRequest) returns (DiagnosticAgentResponse);
 
   // Gather diagnostic information for the running units.
-  rpc DiagnosticUnits(DiagnosticUnitsRequest) returns (DiagnosticUnitsResponse);
+  rpc DiagnosticUnits(DiagnosticUnitsRequest) returns (stream DiagnosticUnitResponse);
```

Comment:
Reminder that we will have to update this in https://github.com/elastic/elastic-agent-client and then update the client version in every Go dependency:
- beats
- cloudbeat
- apm-server (might pick it up as part of automatic updates to libbeat).

It might be worth tracking this outside of the PR on the issue itself.

Reply:
this should not be affected, what we're changing is control server implementation. it's a contract between CLI and agent itself. 
unit diagnostics is not changed aside from increasing a MaxMessageSize limit